runtime/oti/util_api.h

```diff
@@ -1351,6 +1351,27 @@ permittedSubclassesNameAtIndex(U_32* permittedSubclassesCountPtr, U_32 index);
  */
 U_32
 getNumberOfInjectedInterfaces(J9ROMClass *romClass);
+
+/**
+ * Retrieves number of preload classes in this class. Assumes that
+ * ROM class parameter references a class with preloaded classes.
+ *
+ * @param J9ROMClass class
+ * @return U_32 number of preload classes in optionalinfo
+ */
+U_32*
+getNumberOfPreloadClassesPtr(J9ROMClass *romClass);
```

Comment:
The description doesn't quite match the behavior of this function (it doesn't return a number, but rather a pointer to one).